IBPSA/package.mo

```diff
@@ -130,6 +130,15 @@ Non-SI units are to be kept at an absolute minimum, and they must be declared as
   package ReleaseNotes "Release notes"
     extends Modelica.Icons.ReleaseNotes;
 
+  class Version_4_0_0 "Version 4.0.0"
+    extends Modelica.Icons.ReleaseNotes;
+      annotation (preferredView="info", Documentation(info="<html>
+<p>
+Fourth release of the IBPSA library, currently in development status.
+</p>
+</html>"));
+  end Version_4_0_0;
+
```

Comment:
It's nice to use these release notes. However, the current ones don't hold any information. Is that intended?

Reply:
As the IBPSA Library is usually not used directly (all code is in the four derivative libraries), we do not invest the time to enter the release notes. It often is already hard enough to have contributors write good revision notes in the models, which is more important to keep track of why certain changes were done.